qiskit/result/distributions/probability.py

```diff
@@ -13,6 +13,7 @@
 
 import re
 
+
```

Comment:
I'm surprised `black` allows this if it didn't before?

Reply:
Black doesn't care either way, but pycodestyle wants it this way.